api/migrations/0001_initial.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.2.4 on 2023-08-27 14:00
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='countryID',
+            fields=[
+                ('country_id', models.CharField(max_length=3, primary_key=True, serialize=False)),
+                ('country_name', models.CharField(max_length=100)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='pincode',
+            fields=[
+                ('pin_code', models.CharField(max_length=15, primary_key=True, serialize=False, unique=True)),
```

Comment:
please check if pincode is unique across countries as well. If not then you'll need to add a `country_id` to this table as well and the PK will need to be a composite key.